MAUI/DataForm/ManualDataFormItem.md

```diff
@@ -0,0 +1,237 @@
+    items.Add(new DataFormAutoCompleteItem() { FieldName = "Country", ItemsSource = contactInfoViewModel.CountryNames }); 
+    dataForm.AutoGenerateItems = false; 
+    dataForm.Items = items;
+
+{% endhighlight}   
+{% endtabs %} 
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel()
+        {
+            this.ContactDetails = new ContactInfo();
+            this.CountryNames = new List<string>
+            {
+                "United States",
+                "United Kingdom",
+                "France",
+                "Belgium",
+                "Germany"
+            };
+        }
+
+        public ContactInfo ContactDetails { get; set; }
+
+        public IList<string> CountryNames { get; set; }
+
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+{% endhighlight %}
+{% highlight C# %}
+
+    public class ContactInfo
+    {
+        public string Name { get; set; }
+
+        public string Password { get; set; }
+
+        public string Address { get; set; }
+
+        public string Country { get; set; }
+
+        public string Email { get; set; }
+    }
+
+{% endhighlight %}
+
+### Adding DataForm group manually
+You can also generate `DataFormGroupItem` manually using the `Items` property of `SfDataForm`.
+
+{% tabs %}
+{% highlight XAML %}
+    <dataForm:SfDataForm.Items>
+            <dataForm:DataFormGroupItem Name="Address">
+                    <dataForm:DataFormGroupItem.Items>
+                        <dataForm:DataFormMultilineItem FieldName="Address"/>
+                        <dataForm:DataFormTextItem FieldName="State"/>
+                        <dataForm:DataFormTextItem FieldName="ZipCode"/>
+                    </dataForm:DataFormGroupItem.Items>
+        </dataForm:DataFormGroupItem>
+    </dataForm:SfDataForm.Items>
+    
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight C# %}
+
+    DataFormGroupItem groupItem = new DataFormGroupItem();
+        groupItem.Name = "Address";
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "Address" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "State" });
+        groupItem.Items.Add(new DataFormMultilineItem { FieldName = "ZipCode" });
+        this.dataForm.Items.Add(groupItem);
+
+{% endhighlight %}
+{% endtabs %}
+
+{% tabs %}
+{% highlight C# %}
+
+    public class DataFormModel
+    {
+        public string Name { get; set; }
+        public int Phone { get; set; }
+    }
+
+
+    public class DataFormViewModel
+    {
+        public DataFormViewModel
+        {
+            this.DataFormModel = new DataFormModel();
+        }
+
+        public DataFormModel DataFormModel { get; set; }
+        
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`AddCustomItemsimages`
+
+### Dynamically remove manually added dataform items
+You can dynamically remove the dataform items from collections using the `Items` property of `SfDataForm`.
```

Comment:
You can dynamically remove the `DataFormViewItem` from `Items` property of `SfDataForm`.

Reply:
content updated